x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/action/TransportGetRollupIndexCapsAction.java

```diff
@@ -49,6 +56,8 @@ public TransportGetRollupIndexCapsAction(
             ThreadPool.Names.MANAGEMENT
         );
         this.clusterService = clusterService;
+        this.threadPool = threadPool;
```

Comment:
nit: likewise here, I don't think we need to capture this, it's not used AFAICT. Also it's available already as `transportService.getThreadPool()`, no need to add a constructor arg.